package.json

```diff
@@ -7,26 +7,28 @@
   },
   "scripts": {
     "prepare": "husky install",
-    "deploy": "npx hardhat deploy",
-    "lint": "npm run eslint && npm run solhint",
+    "deploy": "npx hardhat deploy --tags Contracts,Poll",
+    "deploy:poll": "npx hardhat deploy --tags Poll",
+    "deploy:contracts": "npx hardhat deploy --tags Contracts",
+    "lint": "yarn eslint && yarn solhint",
     "eslint:fix": "eslint . --ext .js,.ts --fix",
     "eslint": "eslint . --ext .js,.ts",
     "solhint": "npx solhint 'contracts/**/*.sol'",
     "prettier": "npx prettier --write 'contracts/**/*.sol'",
     "clean": "rm -rf cache artifacts typechain",
     "compile": "npx hardhat compile",
     "test:coverage": "npx hardhat coverage",
-    "test:coverage:coveralls": "npm run test:coverage && cat coverage/lcov.info | coveralls",
+    "test:coverage:coveralls": "yarn test:coverage && cat coverage/lcov.info | coveralls",
```

Comment:
We can actually remove this command as we no longer us the `coveralls` npm package to upload coveralls reports. Instead we use the github action https://github.com/marketplace/actions/coveralls-github-action